interfaces/builtin/libvirt.go

```diff
@@ -32,6 +32,7 @@ const libvirtBaseDeclarationSlots = `
 const libvirtConnectedPlugAppArmor = `
 /run/libvirt/libvirt-sock rw,
 /etc/libvirt/* r,
+deny ptrace read peer=snap.@{SNAP_INSTANCE_NAME}.*,
```

Comment:
To stay consistent with other ptrace rules in snapd, can this please be rewritten as follows, and to include a comment as well:

```
# explicitly deny ptrace read for the snap to silence noisy denials.
deny ptrace (read) peer=snap.@{SNAP_INSTANCE_NAME}.@{SNAP_COMMAND_NAME},
```

Also do we know what the particular operation is that is being performed that causes this? Usually it is caused by trying to read /proc/$pid/environ or similar (see `PTRACE_MODE_READ` in `man 2 ptrace`). If we know what this operation is then it would be good to describe that in the comment as well.